libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFDirectoryManager.m

```diff
@@ -235,4 +237,45 @@ - (void)migrateFiles {
     [sharedDefaults synchronize];
 }
 
++ (void)upgradeUserDirectories {
+    [SFDirectoryManager upgradeUserDirectory:NSLibraryDirectory];
+    [SFDirectoryManager upgradeUserDirectory:NSDocumentDirectory];
+}
+
++ (void)upgradeUserDirectory:(NSSearchPathDirectory)type {
```

Comment:
Do we have files on iOS with orgid_userid  in the name (like we do on Android)?

Reply:
Oh, I should have mentioned this as well - we need upgrade steps for the Analytics data as well. The Analytics library takes a path from this library, so you'll need migration steps at both layers. I can talk to you offline about it.

Reply:
I only saw directories with the structure orgId/userId, though realized that I missed profile picture's userId (will add something for that). Is there anywhere else I should check for orgid_userid? Verified that the log files aren't dependent on userId and analytics's event_stores are covered by updating the userId directory in Documents. 
<img width="476" alt="Screen Shot 2020-06-08 at 1 54 02 PM" src="https://user-images.githubusercontent.com/840431/84079523-9301af00-a98f-11ea-9db9-6ec973fcc53e.png">

Reply:
Awesome! I can't think of anything else.